nomt/src/rw_pass_cell/tests.rs

```diff
@@ -0,0 +1,82 @@
+#[test]
+fn test_propagate_writes() {
+    use crate::rw_pass_cell::*;
+
+    loom::model(|| {
+        let domain = RwPassDomain::new();
+        let cell: RwPassCell<u8> = domain.protect(0u8);
+
+        let mut write_pass = domain.new_write_pass();
+        let write_guard = cell.write(&mut write_pass);
+        write_guard.with_mut(|v| *v = 10);
+
+        let write_pass_envelope = write_pass.into_envelope();
+
+        let handle = loom::thread::spawn(move || {
```

Comment:
can you rewrite this test to have 2 threads, where one thread performs the write and sends to the other thread?
I think that the thread spawn boundary already ensures safety.

Reply:
I rewrote it, let me know if I did what you were thinking!